context/src/main/java/io/opentelemetry/context/LazyStorage.java

```diff
@@ -83,6 +85,10 @@ static ContextStorage get() {
 
   static ContextStorage createStorage(AtomicReference<Throwable> deferredStorageFailure) {
     String providerClassName = System.getProperty(CONTEXT_STORAGE_PROVIDER_PROPERTY, "");
+    boolean enforceDefault = Boolean.getBoolean(CONTEXT_STORAGE_PROVIDER_ENFORCE_DEFAULT_PROPERTY);
```

Comment:
Why a second system property? As discussed on the issue, a single one would be simpler.

Reply:
My bad. It's my misunderstand your point. Can you review again? thanks.